service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantAboutToStartControllerTest.java

```diff
@@ -45,7 +43,6 @@
 
     @BeforeEach
     void setup() {
-        given(organisationService.findOrganisation()).willReturn(POPULATED_ORGANISATION);
         given(authTokenGenerator.generate()).willReturn(SERVICE_AUTH_TOKEN);
         given(organisationApi.findOrganisationById(USER_AUTH_TOKEN, SERVICE_AUTH_TOKEN))
```

Comment:
I think that this given can be moved to the other test

Reply:
yeah, done